src/test/java/uk/gov/companieshouse/api/interceptor/TokenPermissionsInterceptorTest.java

```diff
@@ -136,6 +136,10 @@ public void preHandleFeatureFlagOff() throws Exception {
         assertTrue(tokenPermissions.hasPermission(Key.COMPANY_ACCOUNTS, Value.UPDATE));
         assertTrue(tokenPermissions.hasPermission(Key.COMPANY_ACCOUNTS, Value.CREATE));
         assertTrue(tokenPermissions.hasPermission(Key.COMPANY_ACCOUNTS, Value.DELETE));
+        assertTrue(tokenPermissions.hasPermission(Key.COMPANY_STATUS, Value.READ));
+        assertTrue(tokenPermissions.hasPermission(Key.COMPANY_STATUS, Value.UPDATE));
+        assertTrue(tokenPermissions.hasPermission(Key.COMPANY_STATUS, Value.CREATE));
+        assertTrue(tokenPermissions.hasPermission(Key.COMPANY_STATUS, Value.DELETE));
```

Comment:
Can we add assertions for this new permission to the `preHandle` test (with feature flag on)? Possibly adding the new permission to the `AUTHORISED_TOKEN_PERMISSIONS` string to ensure it gets parsed correctly.

Reply:
Additional assertions added.